src/acceptance/README.md

```diff
@@ -50,7 +50,7 @@ The full set of config parameters is explained below:
 
 * `service_name` (required): The name of the registered auto-scaler service, use `cf marketplace` to determine the name.
 * `service_plan` (required): The plan name of the registered auto-scaler service, use `cf marketplace` to determine the plan.
-* `report_interval` (required): How frequently metrics are collected. This value must match the value configured in your deployment.
+* `aggregate_interval` (required): How frequently metrics are aggregated. This value must match the value configured in your deployment.
```

Comment:
aggregate_interval is a deployment setting and can be set less than 60 seconds(should be >= pollInterval in metrics collector).  I saw there is logic in API server that enforce the  StatWindow/BreachDuration to be >= 60 seconds.  This should be changed as long as it is >= aggregate_interval.  

The logic is acceptance test is trying to make it a pass first. The autoscaler needs to have a fix for this, and once we have that, we can remove the following code 

```
interval = cfg.AggregateInterval
if interval < 60 {
    interval = 60
}